R/mrmash_wrapper.R

```diff
@@ -291,30 +273,66 @@ compute_w0 <- function(Bhat, ncomps) {
   return(w0)
 }
 
+
 ### Filter data-driven matrices
```

Comment:
this is not correct roxygen format.

Reply:
This is addressed. The roxygen documentaion for this function `filter_data_driven_mats ` is added